arm/Microsoft.Web/staticSites/readme.md

```diff
@@ -0,0 +1,176 @@
+| `buildProperties` | object | `{object}` |  | Optional. Build properties for the static site. |
+| `cuaId` | string |  |  | Optional. Customer Usage Attribution ID (GUID). This GUID must be previously registered. |
+| `enterpriseGradeCdnStatus` | string | `Disabled` | `[Disabled, Disabling, Enabled, Enabling]` | Optional. State indicating the status of the enterprise grade CDN serving traffic to the static web app. |
+| `location` | string | `[resourceGroup().location]` |  | Optional. Location to deploy static site. The following locations are supported: CentralUS, EastUS2, EastAsia, WestEurope, WestUS2 |
+| `lock` | string | `NotSpecified` | `[CanNotDelete, NotSpecified, ReadOnly]` | Optional. Specify the type of lock. |
+| `name` | string |  |  | Required. Name of the static site. |
+| `privateEndpoints` | array | `[]` |  | Optional. Configuration details for private endpoints. |
+| `provider` | string | `None` |  | Optional. The provider that submitted the last deployment to the primary environment of the static site. |
+| `repositoryToken` | secureString |  |  | Optional. The Personal Access Token for accessing the GitHub repo. |
+| `repositoryUrl` | string |  |  | Optional. The name of the GitHub repo. |
+| `roleAssignments` | array | `[]` |  | Optional. Array of role assignment objects that contain the 'roleDefinitionIdOrName' and 'principalId' to define RBAC role assignments on this resource. In the roleDefinitionIdOrName attribute, you can provide either the display name of the role definition, or its fully qualified ID in the following format: '/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11'. |
+| `sku` | string | `Free` | `[Free, Standard]` | Optional. Type of static site to deploy. |
+| `stagingEnvironmentPolicy` | string | `Enabled` | `[Enabled, Disabled]` | Optional. State indicating whether staging environments are allowed or not allowed for a static web app. |
+| `systemAssignedIdentity` | bool | `False` |  | Optional. Enables system assigned managed identity on the resource. |
+| `tags` | object | `{object}` |  | Optional. Tags of the resource. |
+| `templateProperties` | object | `{object}` |  | Optional. Template Options for the static site. |
+| `userAssignedIdentities` | object | `{object}` |  | Optional. The ID(s) to assign to the resource. |
+
+### Parameter Usage: `buildProperties`
+
+[StaticSiteBuildProperties - Microsoft.Web/staticSites 2021-03-01 - Bicep & ARM template reference | Microsoft Docs](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Web/2021-03-01/staticSites?tabs=bicep#staticsitebuildproperties)
+
+```bicep
+    buildProperties: {
+      apiBuildCommand: 'string'
+      apiLocation: 'string'
+      appArtifactLocation: 'string'
+      appBuildCommand: 'string'
+      appLocation: 'string'
+      githubActionSecretNameOverride: 'string'
+      outputLocation: 'string'
+      skipGithubActionWorkflowGeneration: bool
```

Comment:
Missing the `appsettings` from your original example?
![image](https://user-images.githubusercontent.com/5365358/156942160-e732f1f7-7831-4a8f-9aa6-df1e036cf671.png)

Reply:
Does not seem to be supported: 
https://docs.microsoft.com/en-us/azure/templates/microsoft.web/staticsites?tabs=bicep